.github/workflows/ecosystem-compat.yml

```diff
@@ -47,7 +51,7 @@ jobs:
           path: cocotb-coverage
 
       - name: Install the release version of cocotb-coverage
-        run: pip3 install cocotb-coverage==1.2
+        run: pip3 install --no-deps cocotb-coverage==1.2
```

Comment:
These tools can have more dependencies that just cocotb. I think instead of `--no-deps`, we just move the `cocotb` install below the package-under-test install plus whatever flags are necessary to force the local version to be installed over one gathered from pypi.

It's slightly wasteful, but more manageable.

Reply:
Sure. My idea was to add those manually, when needed, but this will also work.